src/wrapper/signature_wrapper.hpp

```diff
@@ -27,6 +28,8 @@ class signature_wrapper : public wrapper_base<git_signature>
     static author_committer_signatures get_default_signature_from_env(repository_wrapper&);
     static signature_wrapper get_commit_author(const commit_wrapper&);
     static signature_wrapper get_commit_committer(const commit_wrapper&);
+    static signature_wrapper signature_now(std::string name, std::string email);
+    static author_committer_signatures signature_now(std::string author_name, std::string autor_email, std::string committer_name, std::string committer_email);
```

Comment:
Typo in `autor_email`.

Should these be `string_view`? They aren't used yet so I can't tell if this would be a problem or not.